spec/services/prompt_augmentor_spec.rb

```diff
@@ -6,32 +6,34 @@
   let(:chunks) do
     [
       create(:chunk),
-      create(:chunk_from_web),
+      # create(:chunk),
     ]
   end
   let(:search_hits) do
     [
       Search::SearchHit.new(chunks[0], 200.0),
-      Search::SearchHit.new(chunks[1], 220.0),
+      # Search::SearchHit.new(chunks[1], 220.0),
     ]
   end
 
+  # TODO: make part of #prompt test below
+  describe "#prompt for doc from web" do
```

Comment:
Most of the RSpec examples in Archyve (and in general, I think) follow this structure:

```ruby
describe "#prompt" do  # string indicates the instance method `prompt` is being tested by this block
  let(:some_var) { "some_val" }  # something you need to calculate or run before each test, to keep the test clean and/or reuse code
  let(:some_other_var) { "some_other_val" }

  it "has the correct content" do
    expect(subject.prompt).to eq(some_var)
  end
end
```

Reply:
Oh, there is a section for `#prompt` - you can add these examples there!

Reply:
Yeah. I was planning removing the lets and putting it in `#prompt` but  I added this test temporarily to try to figure out why I can't get it to work without the allow_any_instance_of below.